spec/core-1.0.0-rc3.md

```diff
@@ -20,27 +20,6 @@ The pattern of Service Binding has prior art in non-Kubernetes platforms.  Herok
 [osb]: https://www.openservicebrokerapi.org
 [cnb]: https://github.com/buildpacks/spec/blob/master/extensions/bindings.md
 
-<!-- omit in toc -->
```

Comment:
I don't think we can remove this since it is part of the tagged version of the spec (no memory hole-ing). We should remove it from the spec though so it won't be in 1.0.0.

Reply:
I don't really consider this part of "the spec" more so stuff we had in the readme because we didn't have a website. Either way, I can remove it for now and we can revisit

Reply:
The code of conduct link is broken and will need to be fixed at some point, or left as is 🤷